src/components/count-badge/index.tsx

```diff
@@ -0,0 +1,26 @@
+import classNames from 'classnames'
+import Badge from 'components/badge'
+import { CountBadgeProps } from './types'
+import s from './badge-count.module.css'
+
+const CountBadge = ({
```

Comment:
Bike-shedding alert: do you think `NumericBadge` would be a better name? The Count prefix felt confusing when I saw there were semver versions rendered with this component.

Reply:
I agree, `NumericBadge` is a little clearer! I'll make this change. 😊